### PR TITLE
Syntax highlight html in readme

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ Material Design: <a href="http://www.google.com/design/spec/components/card.html
 `paper-card` is a container with a drop shadow.
 
 Example:
-
-    <paper-card heading="Card Title">
-      <div class="card-content">Some content</div>
-      <div class="card-actions">
+```html
+<paper-card heading="Card Title">
+    <div class="card-content">Some content</div>
```
Comment (notwaldorf) [inline README.md]: would you mind only indenting every level by 2 characters, please?

**Comment by notwaldorf** [comment]
Thanks for the fix! I left a small comment (and, if possible, could you squash your commits afterwards? Thanks!)

File: README.md (hunk 1 above)
Comment (samccone) [inline README.md]: sure thing, sorry got a little crazy here :P

**Comment by samccone** [comment]
all set @notwaldorf :)

**Pull request merged**